pandas/tests/extension/test_arrow.py

```diff
@@ -347,6 +362,8 @@ def test_getitem_scalar(self, data):
             exp_type = date
         elif pa.types.is_time(pa_type):
             exp_type = time
+        elif pa.types.is_decimal(pa_type):
+            exp_type = Decimal
```

Comment:
i think we agreed to change/fix the ArrowDtype.type behavior.  since its an API change, let's try to get that in for 2.0